scripts/aberth-init.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import math
+import sys
+
+n = int(sys.argv[1])
+for k in range(n):
+    angle = 2 * math.pi / n
+    off = math.pi / (2 * n)
+    z = angle * k + off
+    c, s = math.cos(z), math.sin(z)
+    print(f"#define K{n}_{k} vec2({c:18.15f}, {s:18.15f})")
```

Comment:
Should part of a separate commit ? Can you add a comment on the purpose of this script ?

Reply:
There are references to that script in the distmap shader (above each batch of defines). I added a comment in the script itself for cross reference. I think they go together so I believe its addition belongs in this commit.